README.md

```diff
@@ -21,14 +22,62 @@ The cluster-insight service does not require any changes to the cluster, and sho
 
 The provided installation script will configure, install, and run cluster-insight. You should be able to run it on any Linux or MacOSX machine. Note, however, that it has only been tested on Ubuntu 14.04 and MacOSX Yosemite.
 
-### Easy installation: run the provided installation script
+### Kubernetes API version
 
-The provided installation script, `cluster-insight-setup.sh`, will install and run the service on a cluster. You can run it as often as you like to update the service (e.g., when the container image or any of the configuration files in the `.install` directory have changed). To run the script:
+The cluster-insight service currently supports only Kubernetes API version "v1".
+To check which API versions are supported by your cluster, run the following command on any machine in your cluster:
```

Comment:
kubectl is only available on the master node of the cluster - so it should be run from there.